<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>730</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: #353538;
border: none;
border-radius: 15px;
border: 1px solid white;
}</string>
    </property>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
       <hintingpreference>PreferDefaultHinting</hintingpreference>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 5px;
background-color:#e0162e;
color:white;
border:none;
}

QPushButton:hover{
background-color:#f24b4b;

}


</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_preview">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton:hover{
           background:white;
            }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_play_pause">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton:hover{
           background:white;
            }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton:hover{
           background:white;
            }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>261</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_lbl">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>180</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="time_curr">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>370</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border:none;
color:white;
}</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_full">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>370</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border:none;
color:white;
}</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nameSound">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
    border:none;
	border:1px solid white;
	border-radius:5px;
	color:white;
}</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMusic">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider{
border:none;
}
QSlider::groove:horizontal{
height: 5px;
background: white;
border-radius:20px;
}
QSlider::handle:horizontal{
background:#777778;
width:3px;
height: 3px;
border-radius:15px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_volume">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>435</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
color: #b8b8ba;
border-color:#b8b8ba;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="sliderVolume">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>445</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider{
border:none;
}
QSlider::groove:horizontal{
height: 5px;
background: white;
border-radius:15px;
}
QSlider::handle:horizontal{
background:#777778;
width:9px;
height: 5px;
border-radius:15px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_thame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
border: 1px solid white;
border-radius:10px;
color:white;
background:black;
}

QPushButton:hover{
background-color: #353538;
}</string>
     </property>
     <property name="text">
      <string>â˜€</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="butt_InfoUser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:none;
border: 1px solid white;
border-radius:10px;
background:white;
}

QPushButton:hover{
background-color: #c2c4c0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>build/Desktop_Qt_6_7_2_MinGW_64_bit-Debug/image/iconUser.png</normaloff>build/Desktop_Qt_6_7_2_MinGW_64_bit-Debug/image/iconUser.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
